.github/workflows/nightly-builds.yml

```diff
@@ -155,7 +155,7 @@ jobs:
             -Dmultinode.Xmx256M \
             -Dmultinode.XX:+AlwaysActAsServerClassMachine \
             ${{ matrix.extraOpts }} \
-            "+~ ${{ matrix.scalaVersion }} clean Test/compile test checkTestsHaveRun"
+            clean "+~ ${{ matrix.scalaVersion }} test" checkTestsHaveRun
```

Comment:
How does the fix work?

Reply:
I don't fully understand yet, but it seems `+~ ${{ matrix.scalaVersion }} foo bar` will run `foo` and `bar` in parallel when the SBT config allows it, and since there's no dependency between `checkTestsHaveRun` and the tests themselves, it would check (and fail) before the tests had finished running.

We might also want to make `checkTestsHaveRun` depend on running the tests, but I wanted to make the build green with this simpler change first :)